nss-day-cohort-43-csharp/Fullstack-Rage-Against-The-Virtual-Machine
# Bl add post tag

# Description
Users can now add tags to their own posts.  However, they cannot be removed yet
Fixes #20 
## Type of change
# Testing Instructions
- In Postman, add a postTag to a post as the owner of the post
- In Postman, try to add a postTag to a post as someone who did not create the post.  Should return unauthorizes
- In postman, try to add a postTag with a bad tagId
- In Postman, try to add a postTag with a bad postId
- In the app, try to add tags to your own posts
- Make sure that adding tags is only an option for the posts of the logged in user
# Checklist:
